lib/rtcpeerconnection/chrome.js

```diff
@@ -329,7 +329,18 @@ ChromeRTCPeerConnection.prototype.createOffer = function createOffer() {
   var options = (args.length > 1 ? args[2] : args[0]) || {};
   var self = this;
 
-  var promise = this._peerConnection.createOffer(options).then(function(offer) {
+  // NOTE(mmalavalli): Starting from Chrome 73, calling createOffer() without
+  // setLocalDescription() on the previous offer will generate new mids for
+  // RTCRtpTransceivers still under negotiation. So, in order to lock in the mids,
+  // we call setLocalDescription() on the pending local offer, if any, before
+  // calling createOffer() on the underlying RTCPeerConnection.
+  var promise = this._sdpSemantics === 'unified-plan' && this._pendingLocalOffer
```

Comment:
How will this affect rollback scenario? Once we apply pendingLocalOffer, we won't be able to rollback. isn't it?